Detectors/HMPID/simulation/src/Detector.cxx

```diff
@@ -429,12 +515,532 @@ TGeoVolume* Detector::createChamber(int number)
   return hmp;
 }
 
+//*************************************************************************************************************
+
+TGeoVolume* Detector::CreateCradle()
```

Comment:
there are many hard-coded numbers here. Is there any chance these can be converted to meaningful names in the future? It would probably ease reading the code (together with a geometry drawing).